frontend/src/App.vue

```diff
@@ -13,6 +13,7 @@ import { Component, Prop, Vue } from "vue-property-decorator";
 import NavBar from "@/components/NavBar.vue";
 import ProgressBar from "@/components/ProgressBar.vue";
 
+
```

Comment:
Do you need this extra space for linting?

Reply:
Updated.